encrypt/openssl.go

```diff
@@ -88,7 +89,7 @@ var (
 	)
 
 	// gets the fingerprint of the private key
```

Comment:
is this really the fingerprint of the private key? Looks like sha256 of the public key. Or is it just convention to use the sha256 of the public key to identify the keypair/private key?

Reply:
it is indeed the sha256 of the public part of the private key. I needed a unique way to identify the key. Since the public key has a 1:1 mapping to the private key I thought this approach was a good idea :-)
Is there a better - maybe even a default - way to compute such a fingerprint? Note that this is only required, internally. But it has to be consistent.

Reply:
Ah, thanks for the explanation. This approach makes sense to me and I don't see any reason to change it.